src/main/resources/db/changelog.xml

```diff
@@ -10,6 +10,9 @@
 
     <property name="identifier_type" value="varchar(63)"/>
     <property name="description_type" value="varchar(2047)"/>
+    <property name="long_description_type" value="text" dbms="postgresql"/>
```

Comment:
do we need the oracle and MSSQL ones too?

Reply:
If we do not intend to support anything other than postgres, we can skip them.  I only added them for completeness when I saw we were including RDBMS specific uuid_type configurations.  Oracle docs recommend clob for this, mssql recommend varchar(max).

Reply:
ah gotcha.  Eh, it doesn't hurt to have it.  We can leave it